### PR TITLE
feat(hints): NewHint#67 cairo_keccak.json `_block_permutation`

### PR DESCRIPTION
**Pull request by Oppen**
## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [x] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
  - [x] CHANGELOG has been updated.

### CODE REVIEW

**Review by MegaRedHand — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,14 +42,15 @@
...
-
     %}
     ```
 
 * Implement hint for `starkware.cairo.common.cairo_keccak.keccak.finalize_keccak` as described by whitelist `starknet/security/whitelists/cairo_keccak.json` [#1041](https://github.com/lambdaclass/cairo-rs/pull/1041)
 
     `BuiltinHintProcessor` now supports the following hint:
 
+* Implement hint for `starkware.cairo.common.cairo_keccak.keccak.finalize_keccak` as described by whitelist `starknet/security/whitelists/cairo_keccak.json` [#1041](https://github.com/lambdaclass/cairo-rs/pull/1041)
+
```
Comment (MegaRedHand) [inline CHANGELOG.md:53]: This entry is duplicated

File: CHANGELOG.md [hunk 2]
```diff
@@ -879,4 +895,4 @@
         * `pub fn from_vm_error(runner: &CairoRunner, error: VirtualMachineError, pc: usize) -> Self` is now `pub fn from_vm_error(runner: &CairoRunner, vm: &VirtualMachine, error: VirtualMachineError) -> Self`
         * `pub fn get_location(pc: &usize, runner: &CairoRunner) -> Option<Location>` is now `pub fn get_location(pc: usize, runner: &CairoRunner) -> Option<Location>`
         * `pub fn decode_instruction(encoded_instr: i64, mut imm: Option<BigInt>) -> Result<instruction::Instruction, VirtualMachineError>` is now `pub fn decode_instruction(encoded_instr: i64, mut imm: Option<&BigInt>) -> Result<instruction::Instruction, VirtualMachineError>`
-        * `VmExcepion` field's string format now mirror their cairo-lang conterparts.
\ No newline at end of file
+        * `VmExcepion` field's string format now mirror their cairo-lang conterparts.
```
Comment (MegaRedHand) [inline CHANGELOG.md:898]:
```suggestion
        * `VmException` fields' string format now mirrors their cairo-lang conterparts.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -912,4 +924,4 @@
         * `pub fn from_vm_error(runner: &CairoRunner, error: VirtualMachineError, pc: usize) -> Self` is now `pub fn from_vm_error(runner: &CairoRunner, vm: &VirtualMachine, error: VirtualMachineError) -> Self`
         * `pub fn get_location(pc: &usize, runner: &CairoRunner) -> Option<Location>` is now `pub fn get_location(pc: usize, runner: &CairoRunner) -> Option<Location>`
         * `pub fn decode_instruction(encoded_instr: i64, mut imm: Option<BigInt>) -> Result<instruction::Instruction, VirtualMachineError>` is now `pub fn decode_instruction(encoded_instr: i64, mut imm: Option<&BigInt>) -> Result<instruction::Instruction, VirtualMachineError>`
-        * `VmExcepion` field's string format now mirror their cairo-lang conterparts.
\ No newline at end of file
+        * `VmException` fields' string format now mirrors their cairo-lang conterparts.
```
Comment (juanbono) [inline CHANGELOG.md:927]: typo in counterparts

**Review by juanbono — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (Oppen) [reply CHANGELOG.md:927]:
```suggestion
        * `VmException` fields' string format now mirrors their cairo-lang counterparts.
```

**Pull request merged**